surf/vendor/edurep/xml_endpoint/v1_2/api.py

```diff
@@ -229,12 +242,13 @@ def _call(self, query, drilldown_names=None,
                                for k, v in parameters.items()])
 
         url = "{}/{}?{}".format(self.api_endpoint, api_method, parameters)
-
         response = requests.get(url)
         if response and response.status_code != requests.codes.ok:
             response.raise_for_status()
-
-        return parse_response(response.text, record_schema=record_schema)
+        if response_parsing:
```

Comment:
Alternative:
`return parse_response(response.text, record_schema=record_schema) if response_parsing else response`
But that's 100% personal preference